lib/registration.pm

```diff
@@ -122,7 +122,7 @@ sub accept_addons_license {
     #   isc co SUSE:SLE-15:GA 000product
     #   grep -l EULA SUSE:SLE-15:GA/000product/*.product | sed 's/.product//'
     # All shown products have a license that should be checked.
-    my @addons_with_license = qw(geo rt idu ids);
+    my @addons_with_license = qw(geo rt idu);
```

Comment:
Why you remove ids? base 1165437 comments  *The displayed name should be "IBM-DLPAR-sdk"* , so there is a bug here but we still need ids.